.github/workflows/e2e-tests-optimized.yml

```
@@ -0,0 +1,640 @@
+    runs-on: ubuntu-latest
+    needs: validate
+    if: needs.validate.outputs.should_run_e2e == 'true'
+    timeout-minutes: 8  # Fixed timeout from 3 to 8 minutes
+    
+    strategy:
+      fail-fast: false
+      max-parallel: ${{ fromJson(needs.validate.outputs.parallel_workers) }}
+      matrix:
+        include: ${{
+          (needs.validate.outputs.browser_matrix == 'standard' && fromJson('[
+            {"browser": "chromium", "browser-name": "Chrome", "category": "desktop", "retry-count": 2},
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 230: mapping values are not allowed in this context

(syntax-check)